apps/els_lsp/src/els_parser.erl

```diff
@@ -49,6 +50,18 @@ forms_to_ast({ok, Forms, _ErrorInfo}) ->
 forms_to_ast({error, _ErrorInfo} = Error) ->
   Error.
 
+-spec parse_incomplete_text(string(), {erl_anno:line(), erl_anno:column()})
```

Comment:
Isn't this unused?

Reply:
its only used for manual testing (similar to parse_text and parse_file) I found them useful while developing. I should add a comment to clarify.

Reply:
I see. Fair enough, let's add a comment to clarify.

Reply:
added a comment in the export section